Add/organize introductions to guidelines chapters (#3855)
qgis/QGIS-Documentation: source/docs/documentation_guidelines/index.rst

Documentation Guidelines
========================

QGIS Documentation will be built automatically on the server at 0, 8am, 4pm
US/Pacific (Pacific Time). The current status is available at
https://docs.qgis.org.

QGIS Documentation source files are available at https://github.com/qgis/QGIS-Documentation.
They are mainly written using the reStructuredText (reST) format syntax, coupled
with some scripts from the Sphinx toolset to post-process the HTML output.
For general information on these tools, see
http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html or
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html.

The following chapters will guide you through learning:

* how to manage the documentation source files using `git <https://git-scm.com/>`_
  system and the `GitHub <https://github.com/>`_ platform on which they are stored
* how to modify the texts, provide screenshots... in a compliant way
* how to share and ensure your modifications are pushed to the official docs.

> If you are looking for general rules on contributing to QGIS project or managing

Reviewer: ```suggestion
If you are looking for general information about how to contribute to the QGIS project or manage
```

Author: Actually, I'd remove the mention of repository management here, focusing only on "you need help, contact the list"

Author: ```suggestion
If you are looking for general information about how to contribute to the QGIS
```

> repositories, you may find help at `Get Involved in the QGIS Community

Author: ```suggestion
project, you may find help at `Get Involved in the QGIS Community
```
